src/jdk.jdeps/share/man/jnativescan.1

```diff
@@ -0,0 +1,215 @@
+.TP
+\f[V]--class-path\f[R] \f[I]path\f[R]
+Used to specify a path list of class path jar files to be scanned.
+All jar files specified through this path will be scanned.
+If a jar file contains a \f[V]Class-Path\f[R] attribute in its manifest,
+jar files listed there will be scanned as well.
+Jar files listed in the \f[V]Class-Path\f[R] manifest attribute that can
+not be found are ignored.
+.TP
+\f[V]--module-path\f[R] \f[I]path\f[R]
+Used to specify a path list of jar files or directories containing
+module path jar files.
+The list jar files that will be scanned depends on the
+\f[V]--add-modules\f[R] option.
+.RS
+.PP
+For both the \f[V]--class-path\f[R] and \f[V]--module-path\f[R] options,
+\f[I]path\f[R] should be a search path that consists of one or more jar
+files, separated by the system-specific path separator.
+For example:
+.IP \[bu] 2
+\f[B]Linux and macOS:\f[R]
+.RS 2
+.RS
+.PP
+\f[V]--class-path /some/foo.jar:/another/different/bar.jar\f[R]
+.RE
+.RE
+.PP
+\f[B]Note:\f[R]
+.PP
+On Windows, use a semicolon (\f[V];\f[R]) as the separator instead of a
+colon (\f[V]:\f[R]).
+.IP \[bu] 2
+\f[B]Windows:\f[R]
+.RS 2
+.RS
+.PP
+\f[V]--class-path C:\[rs]some\[rs]foo.jar;C:\[rs]another\[rs]different\[rs]bar.jar\f[R]
+.RE
+.RE
+.RE
+.TP
+\f[V]--add-modules\f[R] \f[I]module[,module...]\f[R]
+Used to specifiy a comma-separated list of module names that indicate
+the root modules to scan.
+All the modules in the root set will be scanned, as well as any modules
+that they depend on.
+This includes dependencies on service implementations specified through
+the \f[V]uses\f[R] directive in a module\[aq]s \f[V]module-info\f[R]
+file.
+All modules found on the module path that provide an implementation of
+such a service will be scanned as well.
+.TP
+\f[V]--release\f[R] \f[I]version\f[R]
+Used to specify the Java SE release that specifies the set of restricted
+methods to scan for.
+For multi-release jar files, this option also indicates the version of
+class file that should be loaded from the jar.
+This option should be set to the version of the runtime under which the
+application is eventually intended to be run.
+If this flag is omitted, the version of \f[V]jnativescan\f[R] is used as
+release version.
+.TP
+\f[V]--print-native-access\f[R]
+Print a comma-separated list of module names that use native
+functionalities, instead of the default tree structure.
+.TP
+\f[V]--help\f[R] or \f[V]-h\f[R]
+Prints out a full help message.
+.TP
+\f[V]--version\f[R]
+Prints out the abbreviated version string of the tool.
+.SH EXAMPLE OF \f[V]jnativescan\f[R] USE
+.PP
+\f[V]jnativescan\f[R] accepts a runtime configuration in the form of a
+class path, module path, set of root modules, and a target release
+version.
+For the class path, the tool will scan all jar files, including those
+found recursively through the \f[V]Class-Path\f[R] manifest attribute.
+For the module path, the tool scans all modules in the root module set
```

Comment:
Note: sometimes you say root modules, sometimes root module set. I'd suggest to pick one and remain consistent.